exercises/02_sort3/sort3.go

```diff
@@ -2,7 +2,17 @@ package main
 
 import "fmt"
 
+func minmax(a, b int) (min, max int) {
```

Comment:
The tests for this program do not pass: https://github.com/prog-1/functions/runs/4202969859?check_suite_focus=true.